snippets/base/fields.py

```diff
@@ -10,3 +10,7 @@ def __init__(self, *args, **kwargs):
                   'validators': [validate_regex]}
         myargs.update(kwargs)
         return super(RegexField, self).__init__(*args, **myargs)
+
+
+class URLField(models.CharField):
```

Comment:
Why can't you use Django's built-in `URLField`?

Reply:
Oh, I missed that it was your custom validator. I guess there's no real advantage to having this be a subclass of Django's `URLField`. 👍

Reply:
You're correct!